app/views/dashboard/_supervisors_table.html.erb

```diff
@@ -23,15 +23,12 @@
         <td id="name">
           <%= link_to(supervisor.display_name, edit_supervisor_path(supervisor)) %>
         </td>
-        <td id="volunteer-assignments"> <%= supervisor.volunteers.count %></td>
+        <td id="volunteer-assignments"> <%= supervisor.volunteers.size %></td>
         <td id="serving-transition-aged-youth">
-          <%= supervisor.volunteers.select { |v| v.serving_transition_aged_youth? }.count %>
+          <%= supervisor.volunteers.includes(:casa_cases).where(casa_cases: {transition_aged_youth: true}).size %>
         </td>
         <td id="no-contact">
-          <%=
-            days_since_contact = 14
-            supervisor.volunteers.select { |v| v.recent_contacts_made(days_since_contact) < 1 }.count
-          %>
+          <%= supervisor.volunteers.includes(:case_contacts).where(case_contacts: {contact_made: true}).where.not(case_contacts: { occurred_at: 14.days.ago..Date.today}).size %>
```

Comment:
Very nice! I'd like to eventually move query logic like this and the above into the controller and make the views less smart